### PR TITLE
add new flockos java sdk (generated by swagger)

### PR DESCRIPTION
**Pull request by amnsinghl**

### CODE REVIEW

**Comment by chaitanyagupta** [comment]
Please add a README

**Comment by chaitanyagupta** [comment]
I see a README, but I don't think its updated.

**Comment by amnsinghl** [comment]
OK will update

**Comment by amnsinghl** [comment]
@chaitanyagupta I have updated readme in	67ea7d9

File: gradlew [hunk 1]
```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env bash
```
Comment (ajitd) [inline gradlew:1]: Do we need this file to be part of the repo?

File: gradlew.bat [hunk 2]
```diff
@@ -0,0 +1,90 @@
+@if "%DEBUG%" == "" @echo off
+@rem ##########################################################################
+@rem
```
Comment (ajitd) [inline gradlew.bat:3]: Is this file needed?

File: src/main/java/co/flock/api/Roster.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+import java.util.HashMap;
+import java.util.Map;
+
+public class Roster {
+
+    /**
+     *  (makes an asynchronous HTTP request)
+     * 
+     * @param token  (required) 
+     * @return List&lt;PublicProfile&gt;
```
Comment (ajitd) [inline src/main/java/co/flock/api/Roster.java]: &lt; looks a bit unreadable, we should remove escaping

File: src/main/java/co/flock/model/ClientFlockmlAction.java [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+  }
+
+  @Override
+  public int hashCode() {
+    return Objects.hash(userName, chat, chatName, messageId, messageUid, attachmentId, actionId, super.hashCode());
+  }
+
+
+  @Override
+  public String toString() {
```
Comment (ajitd) [inline src/main/java/co/flock/model/ClientFlockmlAction.java]: There should be hava javadoc on this method so that a dev can figure out that he needs to use toString to get the json for this object
Also we should add another function toJsonString in the swagger generator files which does the same thing.

File: gradlew (hunk 1 above)
Comment (amnsinghl) [reply gradlew:1]: this file is generated when you run `gradle init`
Also gradle recommends using `gradlew` as it forces the user to use a particular version of gradle while building the project

File: gradlew.bat (hunk 2 above)
Comment (amnsinghl) [reply gradlew.bat:3]: Yes
same reason as for `gradlew`

File: src/main/java/co/flock/api/Roster.java (hunk 3 above)
Comment (amnsinghl) [reply src/main/java/co/flock/api/Roster.java]: fixed in 52045ef

File: src/main/java/co/flock/model/ClientFlockmlAction.java (hunk 4 above)
Comment (amnsinghl) [reply src/main/java/co/flock/model/ClientFlockmlAction.java]: fixed in 	2a6fde5
